docs/resources/service_account_application_key.md

```diff
@@ -0,0 +1,45 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "datadog_service_account_application_key Resource - terraform-provider-datadog"
+subcategory: ""
+description: |-
+  Provides a Datadog ServiceAccountApplicationKey resource. This can be used to create and manage Datadog Service Account Application Keys.
+---
+
+# datadog_service_account_application_key (Resource)
+
+Provides a Datadog ServiceAccountApplicationKey resource. This can be used to create and manage Datadog Service Account Application Keys.
```

Comment:
```suggestion
Provides a Datadog ServiceAccountApplicationKey resource. This can be used to create and manage Datadog service account application keys.
```